docs/source/whatsnew_1_4.md

```diff
@@ -0,0 +1,60 @@
+# What's new in 1.4 🎉🎉
+
+- MAISI: state-of-the-art 3D Latent Diffusion Model
+- VISTA3D: interactive foundation model for segmenting and anotating human anatomies
```

Comment:
Update to VISTA-3D and VISTA-2D.  Same for below in headers and description for the models.

Reply:
Updated, thanks.